demo-notebooks/guided-demos/pytorch_lightning.py

```diff
@@ -0,0 +1,98 @@
+import os
+import tempfile
+
+import torch
+from torch.utils.data import DataLoader, DistributedSampler
+from torchvision.models import resnet18
+from torchvision.datasets import FashionMNIST
+from torchvision.transforms import ToTensor, Normalize, Compose
+import lightning.pytorch as pl
+
+import ray.train.lightning
+from ray.train.torch import TorchTrainer
+
+# Based on https://docs.ray.io/en/latest/train/getting-started-pytorch-lightning.html
+
+"""
+Note: This example requires an S3 compatible storage bucket for distributed training. Please visit our documentation for more information -> https://github.com/project-codeflare/codeflare-sdk/blob/main/docs/s3-compatible-storage.md
```

Comment:
How do I configure what bucket to actually use for the distributed training?

Reply:
I created my own bucket with its own path via AWS and gathered the URI using the UI. 
`s3://mark-bucket/data/`

I was not aware we had a shared bucket but you could create a new folder within it and then copy the URI from there.